tests/regression.rs

```diff
@@ -97,6 +97,14 @@ fn test_rotate_nearest_rgb() {
     compare_to_truth_rgb("elephant.png", "elephant_rotate_nearest.png", rotate_nearest_about_center);
 }
 
+#[test]
+fn test_rotate_nearest_rgba() {
+    fn rotate_nearest_about_center(image: &RgbaImage) -> RgbaImage {
+        rotate_about_center(image, std::f32::consts::PI/4f32, Interpolation::Nearest)
+    }
+    compare_to_truth_rgb("elephant_rgba.png", "elephant_rotate_nearest_rgba.png", rotate_nearest_about_center);
+}
```

Comment:
This doesn't look like it would work - you're calling `compare_to_truth_rgb` with a function on `RgbaImage`s. I think you need to add a `compare_to_truth_rgba` method.